### DISCUSSION TITLE
Testing the Waters

### DISCUSSION THREAD

**Discussion by randybhill** [keiwando/toonsquid#289] [Show and tell]
Trying out the Squid for a couple weeks now.  Been working with FlipaClip and Cartoon Animator 4.  Not real happy with them go strictly Microsoft.  Very happy with ToonSquid so far.  Total novice here. Getting started a little late in life but enjoying creating.  

Here’s a little montage of some random ideas I’ve been putting together. 

https://youtu.be/3h3z9uNtPGw

**Comment by Cat-Butler** [comment]
Nice usage of the features! Good to see a small storyline with Vinnie (or should that be pronounced Vinie? 😺).
A fairly small community so far, but I hope we'll grow.

For the desktop, I prefer Moho to Cartoon Animator (Moho Debut is not a bad bargain) - a little more manual rigging work but greater potential.
Hoping Keiwan will introduce more rigging features into ToonSquid (as far as I'm concerned, there is already a sufficiency of frame-by-frame apps) but, as you've shown, there are already enough features to work with.

**Comment by MojoMan49** [comment]
**So, the rabbit is my first usage of 1.90 using a symbol imported from another library**
![IMG_2170](https://github.com/keiwando/toonsquid/assets/127110520/e00ed84d-6be9-4a96-9e63-1dc099664a3d)
**This next one of Mark: 21-27** (_Really not trying to advocate for religious belief here_) also incorporates symbols. I’m still not too sure about what I call nesting of layers within a symbol at the point where one can **_“edit clip”_** and using additional layers and then markers create a sort of self contained character, but I’m sure trying
https://github.com/keiwando/toonsquid/assets/127110520/369a2353-be52-4747-9146-fa19ef7926f3
